python-sdk/nuimages/nuimages.py

```diff
@@ -201,6 +201,31 @@ def check_sweeps(self, filename: str) -> None:
                                 'directory, but you cannot call methods that use non-keyframe sample_datas.'
                                 % sweeps_dir)
 
+    def find_corresponding_sample_data(self, sd_token: str, tgt_modality: str) -> str:
+        """
+        For a sample_data token from either camera or lidar, find the corresponding sample_data token of the
+        other modality.
+        :param sd_token: Source sample_data token.
+        :param tgt_modality: The modality of the target.
+        :return: The tcorresponding sample_data token with the targe modality.
```

Comment:
`tcorresponding` -> typo here?

Reply:
And `targe` as well (guessing it's supposed to be `target`)

Reply:
The total number of Ts is correct then :-D

Reply:
:joy: